static/app/types/sampling.tsx

```diff
@@ -147,6 +147,7 @@ export type UniformModalsSubmit = (props: {
 export enum DynamicSamplingBiasType {
   BOOST_ENVIRONMENTS = 'boostEnvironments',
   BOOST_LATEST_RELEASES = 'boostLatestRelease',
+  BOOST_KEY_TRANSACTION = 'boostKeyTransaction',
```

Comment:
nit: should this be plural like rest of them?

Reply:
it's mix `Environments` is plural; `Release` not :(

Reply:
yes, thought about it too...but at least all are plural in the frontend

Reply:
I meant enum key - BOOST_KEY_TRANSACTIONS

Reply:
make sense i'm using same in py part: `BOOST_KEY_TRANSACTIONS_RULE`